public/style.css

```diff
@@ -138,10 +138,10 @@ input.nav_search_box{
 .num_count {
   font-family: var(--quiz-title-font);
   font-size: 20px;
-  position: absolute;
+  float: right;
   top: 0px;
   right: 50px;
-  color: var(--darkgrey);
+  color: var(--light-text);
```

Comment:
Can you split the `preview` css for `.num_count` into a different class? This change will break the quiz page.